element-hq/element-x-android
# Implement MSC2530

Depends on https://github.com/matrix-org/matrix-rust-sdk/pull/3171
Fixes https://github.com/element-hq/element-x-android/issues/2521
## Type of change


## Content

Show body as image caption, per MSC 2530

## Motivation and context

Caption as a different message doesn't look this good

## Screenshots / GIFs
![image](https://github.com/element-hq/element-x-android/assets/683652/b0dede01-d9d2-4c61-b8c5-d40ae18c1f21)



## Tests



- Send an image using with a bridge/client with body as caption enabled (mautrix-whatsapp for example)
- See how the image is displayed with caption instead of in a separate message.


## Tested devices

- OS version(s):

## Checklist
